<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppInfoWidget</class>
 <widget class="QWidget" name="AppInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AppInfoWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="currentIndex">
      <number>6</number>
     </property>
     <widget class="QWidget" name="info_page">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="margin">
        <number>9</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="picture_frame_layout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="picture_frame">
           <property name="minimumSize">
            <size>
             <width>240</width>
             <height>175</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>240</width>
             <height>175</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_7"/>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="info_form_layout">
         <item row="0" column="0">
          <widget class="QLabel" name="name_label">
           <property name="text">
            <string>Application Name</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="version_label">
           <property name="text">
            <string>Current Version</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="organizer_label">
           <property name="text">
            <string>Organizer/Company</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="copyright_label">
           <property name="text">
            <string>Copyright</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="description_label">
           <property name="text">
            <string>Description</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QTextBrowser" name="description_text"/>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="name_line">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="version_line">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="organizer_line">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="copyright_line">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="about_page">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="margin">
        <number>9</number>
       </property>
       <item>
        <widget class="QGroupBox" name="about_box">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>About</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTextBrowser" name="about_text"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="command_box">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Commands</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QTextBrowser" name="command_text"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="details_page">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="margin">
        <number>9</number>
       </property>
       <item>
        <widget class="QGroupBox" name="depends_box">
         <property name="title">
          <string>Depends</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTreeWidget" name="depends_list">
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <property name="itemsExpandable">
             <bool>true</bool>
            </property>
            <property name="animated">
             <bool>true</bool>
            </property>
            <property name="headerHidden">
             <bool>true</bool>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="headerStretchLastSection">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Type</string>
             </property>
            </column>
            <item>
             <property name="text">
              <string>Binary Depends</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>SApplication Depends</string>
             </property>
             <property name="icon">
              <iconset resource="resource.qrc">
               <normaloff>:/files/Pixs/Silicon.png</normaloff>:/files/Pixs/Silicon.png</iconset>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="other_details_form">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::ExpandingFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="multi_run_label">
           <property name="text">
            <string>Allow MultiRun</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="multi_run_line">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="type_label">
           <property name="text">
            <string>Application Type</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QListWidget" name="type_list"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="authors_page">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="margin">
        <number>9</number>
       </property>
       <item>
        <widget class="QGroupBox" name="authors_box">
         <property name="title">
          <string>Authors</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="contact_page">
      <layout class="QFormLayout" name="formLayout_2">
       <property name="margin">
        <number>9</number>
       </property>
       <item row="2" column="0">
        <widget class="QLabel" name="home_page_label">
         <property name="text">
          <string>Home Page</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="home_page_line">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="mail_label">
         <property name="text">
          <string>EMail</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="mail_line">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="license_page">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="margin">
        <number>9</number>
       </property>
       <item>
        <widget class="QGroupBox" name="license_box">
         <property name="title">
          <string>License</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTextBrowser" name="license_text"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="translators_page">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Special thanks to translators ...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="translators_table">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="dragDropOverwriteMode">
          <bool>false</bool>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
         <property name="gridStyle">
          <enum>Qt::DotLine</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>173</number>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>153</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Country</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Language</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Translator Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>HomePage</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Email</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
